core/src/components/datetime-button/test/a11y/datetime-button.e2e.ts

```diff
@@ -0,0 +1,47 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+/**
+ * The tested behavior does not vary across directions
+ */
+configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
```

Comment:
datetime-button doesn't have per-mode styles, so we can skip one of the modes
```suggestion
configs({ directions: ['ltr'], modes: ['ios'] }).forEach(({ title, screenshot, config }) => {
```

Reply:
Also if you merge this you'll need to manually delete the redundant generated screenshots